### ISSUE TITLE
run.sh script should use maven wrapper

### ISSUE THREAD

**Issue by benvanscheppingen** [spring-guides/gs-crud-with-vaadin#66]
run.sh shows the following message:

[ERROR]
Downloading: http://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-junit-platform/2.22.2/surefire-junit-platform-2.22.2.pom
Return code is: 501 , ReasonPhrase:HTTPS Required

I searched, found and followed the instructions: https://blog.adamgamboa.dev/2020/01/16/maven-central-repository-is-moving-to-https-and-disablig-http-access/

According these instructions I changed pom.xml in both maps (complete and initial)  but the above error still occurs.

What is it I am doing wrong?  
Thanks

**Comment by dsyer** [comment]
The run.sh script doesn’t use the maven wrapper (it should). So you are picking up an older version of mvn from your local path.

**Comment by benvanscheppingen** [comment]
Thanks.
Checked Maven (at command prompt where I run the shell script):

[ben@localhost complete]$ mvn -v
Apache Maven 3.2.2 (NON-CANONICAL_2015-04-01T06:56:20_mockbuild; 2015-04-01T08:56:20+02:00)
Maven home: /usr/share/maven

It helped already when I added the repository annotation, the run.sh did much better. Except  the "surefire" part. Is it possible that "surefire" doesn't use then complete/pom.xml and initial/pom.xml  but a settings.xml from anywhere else?
